Gemfile

```diff
@@ -5,4 +5,4 @@ gem 'nokogiri', '= 1.6.8.1'
 gem 'rubyXL'
 
 gem 'minitest'
-gem 'minitest-ci', git: 'git@github.com:circleci/minitest-ci.git'
+gem 'minitest-ci', git: 'https://github.com/circleci/minitest-ci.git', branch: 'master'
```

Comment:
@nllong This is in your area of expertise. Is this change OK with you?

Reply:
I am exclusively a git@ protocol user, so I'm not sure the impact of this. Based on the number of warnings I have been getting around using `https`, I'm sure this is okay.